examples/test/test_i_one_kms_cmk.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Unit test suite for the Strings examples in the AWS-hosted documentation."""
```

Comment:
This is inaccurate. A reasonable description of this would be to identify the examples file that this test tests.

Reply:
I will make all the above changes. Thank you!